api/v1/app.py

```diff
@@ -7,16 +7,17 @@
 
 app = Flask(__name__)
 app.config['JSONIFY_PRETTYPRINT_REGULAR'] = True
-
+app.url_map.strict_slashes = False
 app.register_blueprint(app_views)
 
+
 @app.teardown_appcontext
 def teardown(self):
     storage.close()
 
+
 if __name__ == "__main__":
     app.run(
         host=os.getenv('HBNB_API_HOST'),
         port=os.getenv('HBNB_API_PORT'),
-        threaded=True)
-    
+        threaded=True)
```

Comment:
Don't have newline at the end of file

Reply:
Pycodestyle must the file dont have new line at the end of file